kas/machine/jetson-orin-nano-devkit-nvme.yml

```diff
@@ -18,7 +18,5 @@ repos:
 
 local_conf_header:
   machine-avocado-jetson-orin-nano-devkit-nvme: |
-    TEGRA_OOT_ALL_DRIVER_PACKAGES:append = " nv-kernel-module-rtl8852ce"
```

Comment:
No longer needed? Or is this appended someplace else?

Reply:
This was a bug and should have never been here. It was causing a duplicate package in the build